web/controllers/organization_controller.ex

```diff
@@ -0,0 +1,55 @@
+defmodule CodeCorps.OrganizationController do
+  use CodeCorps.Web, :controller
+
+  alias CodeCorps.Organization
+  alias JaSerializer.Params
+
+  plug :scrub_params, "data" when action in [:create, :update]
+
+  def index(conn, _params) do
+    organizations = Repo.all(Organization)
+    organizations = Repo.preload(organizations, [:slugged_route])
+    render(conn, "index.json-api", data: organizations)
+  end
+
+  def create(conn, %{"data" => data = %{"type" => "organization", "attributes" => _organization_params}}) do
+    changeset = Organization.create_changeset(%Organization{}, Params.to_attributes(data))
+
+    case Repo.insert(changeset) do
+      {:ok, organization} ->
+        organization = Repo.preload(organization, [:slugged_route])
+
+        conn
+        |> put_status(:created)
+        |> put_resp_header("location", organization_path(conn, :show, organization))
+        |> render("show.json-api", data: organization)
+      {:error, changeset} ->
+        conn
+        |> put_status(:unprocessable_entity)
+        |> render(CodeCorps.ChangesetView, "error.json-api", changeset: changeset)
+    end
+  end
+
+  def show(conn, %{"id" => id}) do
+    organization = Repo.get!(Organization, id)
+    organization = Repo.preload(organization, [:slugged_route])
+    render(conn, "show.json-api", data: organization)
+  end
+
+  def update(conn, %{"id" => id, "data" => data = %{"type" => "organization", "attributes" => _organization_params}}) do
+    organization = Repo.get!(Organization, id)
+    organization = Repo.preload(organization, [:slugged_route])
```

Comment:
```Elixir
changeset = 
  Repo.get(Organization, id)
  |> Repo.preload([:slugged_route])
  |> Organization.changeset(Params.to_attributes(data))
```

This would be more elixir-like, no? Same goes for other instances in this file.